### DISCUSSION TITLE
What is the best way to determine if requested URL is related to admin pages/resources?

### DISCUSSION THREAD

**Discussion by arkadiuszwojcik** [OrchardCMS/OrchardCore#16672] [Q&A] [answered]
I am writing my own module for redirection, but I would like just in case exclude all urls related to admin pages. What is the best way to check in OC if given URL (for current tenant) is admin related or is it regular one?

**Comment by hishamco** [comment, marked as answer]
Simple you can check the URL if contains `AdminUrlPrefix` in the `OC.Admin` module, for more info check https://docs.orchardcore.net/en/latest/reference/modules/Admin/
